javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,69 @@
+//Movies exercise
+
+const apiUri = `https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json`;
+
+async function movies() {
+  const response = await fetch(apiUri);
+  const dataMovies = await response.json();
+  console.log(dataMovies);
+
```

Comment:
Nitpick:
While it's nice to use tracing when writing your code, it's good practice to remove them before committing the code.